test/integration/targets/azure_rm_keyvault/tasks/main.yml

```diff
@@ -82,12 +82,27 @@
         keys:
           - get
           - list
+    tags:
+      aaa: bbb
   register: output
 - name: Assert the state has changed
   assert:
     that:
       - output.changed == true
 
+- name: Use REST API to retrieve keyvault (as facts not available yet)
```

Comment:
doesn't show in the module return value?

Reply:
no, this is one of the new modules, and the rule is not to return input information.